docs/yul.rst

```diff
@@ -193,6 +193,10 @@ has to be specified after a colon:
 
     let x := and("abc":uint32, add(3:uint256, 2:uint256))
 
+The 32-character limit does not apply to string literals passed to builtin functions that require
```

Comment:
Can you move this closer to where the limit is mentioned?

Reply:
How about now? I moved it up and merged with last sentence of the paragraph mentioning the limit.